src/components/Card.tsx

```diff
@@ -1,7 +1,11 @@
+import { useState } from 'react';
 import { History, MurojaahType } from '@/api/module/murojaah/entity';
 import { formatDatetime } from '@/util/datetime';
+import { clsx } from 'clsx';
 
 export const Card = (item: History): JSX.Element => {
+  if (!Object.values(MurojaahType).includes(item.murojaahType)) return <></>;
```

Comment:
can you move this inside the useMemo in the bottom? (https://github.com/RR-Ardhika/murojaah/pull/9/files#r1502141160)

Reply:
removed it coz just use default case in https://github.com/RR-Ardhika/murojaah/pull/9/commits/cde885b5ea252ff7b23d0d6d9c4a9eb1a08ea4a9